guides/json.md

```diff
@@ -0,0 +1,52 @@
+- [Making Raw JSON REST Requests](#making-raw-json-rest-requests)
+  - [GET](#get)
+  - [PUT](#put)
+  - [POST](#post)
+  - [DELETE](#delete)
+
+# Making Raw JSON REST Requests
+OpenSearch exposes a REST API that you can use to interact with OpenSearch. The OpenSearch .NET client provides a low-level API that allows you to send raw JSON requests to OpenSearch. This API is useful if you want to use a feature that is not yet supported by the OpenSearch .NET client, but it supported by the OpenSearch REST API.
```

Comment:
You can link to the sample code from here, like the python guide does.

Additionally I think it'd make more sense to use the same example actions that the python guide does: https://github.com/opensearch-project/opensearch-py/blob/main/guides/json.md
https://github.com/opensearch-project/opensearch-py/blob/main/samples/json/hello.py